pkg/types/csv_rbac.go

```diff
@@ -0,0 +1,263 @@
+package types
+
+import (
+	"fmt"
+	"sort"
+)
+
+type CsvPermissions struct {
```

Comment:
Any reason why we use custom types instead of reusing the `operator-framework/api` types? https://github.com/operator-framework/api/tree/master/pkg/operators/v1alpha1

Reply:
I thought of creating a new type so that we can add new attributes in the future if needed. But like you said, wrapping the `operator-framework/api` type inside this custom struct also works. Will refactor this PR.